src/moisture_reading.py

```diff
@@ -13,16 +14,26 @@ def create_table(cursor = c):
                    datetime TEXT
                    )""")
 
-# @eel.expose
-def create(reading, current_time, cursor = c):
+@eel.expose
+def create(reading, time_class = datetime, cursor = c):
     with conn:
-        datetime_string = current_time.strftime('%Y-%m-%d %H:%M:%S')
+        datetime_string = time_class.now().strftime('%Y-%m-%d %H:%M:%S')
         cursor.execute(f"INSERT INTO readings(reading, datetime) VALUES ({reading}, '{datetime_string}')")
         query = f'SELECT * FROM readings WHERE id={cursor.lastrowid}'
         res = cursor.execute(query)
         return res.fetchall()[0]
-#
-# @eel.expose
+
+@eel.expose
 def all(cursor = c):
     cursor.execute('SELECT * FROM readings')
     return cursor.fetchall()
+
+
+@eel.expose
+def format_readings(cursor = c):
```

Comment:
Should this be extracted to another class if it is not directly affecting the database?

Reply:
It needs to be outside of a class for eel - it cannot parse Python classes on the JS side

Reply:
Makes sense, maybe we should add a comment to that effect to explain?